src/PowerShellEditorServices/Services/PowerShell/Debugging/DscBreakpointCapability.cs

```diff
@@ -91,10 +91,12 @@ public static async Task<DscBreakpointCapability> GetDscCapabilityAsync(
                     .AddCommand(@"Microsoft.PowerShell.Core\Import-Module")
                     .AddParameter("Name", "PSDesiredStateConfiguration")
                     .AddParameter("PassThru")
-                    .AddParameter("ErrorAction", ActionPreference.Ignore);
+                    .AddParameter("ErrorAction", ActionPreference.Ignore)
+                    .AddCommand(@"Microsoft.PowerShell.Utility\Select-Object")
+                    .AddParameter("ExpandProperty", "Name");
 
-                IReadOnlyList<PSModuleInfo> dscModule =
-                    await executionService.ExecutePSCommandAsync<PSModuleInfo>(
+                IReadOnlyList<string> dscModule =
+                    await executionService.ExecutePSCommandAsync<string>(
```

Comment:
Ok so these are now strings since we added the parameter to get the names instead of the modules directly. Why, what are we changing here? FWIW this is really old legacy code.